.gitignore

```diff
@@ -3,3 +3,4 @@
 .DS_Store
 public/
 resources/
+.idea
```

Comment:
What is .idea :) just curious ;)

Reply:
It's to ignore files created by "IntelliJ IDEA" (https://www.jetbrains.com/idea/) which is the integrated development environment I am using :)

Reply:
Oh gotcha :)